src/core/interpreter_test.cc

```diff
@@ -526,4 +528,54 @@ TEST_F(InterpreterTest, LuaIntOverflow) {
   EXPECT_FALSE(Execute("EVAL \"struct.pack('>I2147483648', '10')\" 0"));
 }
 
+TEST_F(InterpreterTest, LuaGcStatistic) {
+  InterpreterManager im(1);
+  auto* interpreter = im.Get();
+
+  std::string_view keys[] = {"key1", "key2", "key3", "key4", "key5", "key6", "key7"};
+  interpreter->SetGlobalArray("KEYS", SliceSpan{keys});
+
+  auto cb = [](Interpreter::CallArgs ca) {
+    auto* reply = ca.translator;
+    reply->OnInt(1);
+  };
+  interpreter->SetRedisFunc(cb);
```

Comment:
It's a good test but since it's not straightforward to understand - please add comments what this script does